setup.py

```diff
@@ -30,18 +30,88 @@ def read_version_py(infname):
 VERSION_PY_FILENAME = 'src/allmydata/_version.py'
 version = read_version_py(VERSION_PY_FILENAME)
 
-# Tahoe's dependencies are managed by the find_links= entry in setup.cfg and
-# the _auto_deps.install_requires list, which is used in the call to setup()
-# below.
-adglobals = {}
-auto_deps_fn = "src/allmydata/_auto_deps.py"
-if sys.version_info[0] >= 3:
-    exec(compile(open(auto_deps_fn, 'rb').read(), auto_deps_fn, "exec"),
-         adglobals, adglobals)
-else:
-    execfile(auto_deps_fn, adglobals)
-install_requires = adglobals['install_requires']
-setup_requires = adglobals['setup_requires']
+install_requires = [
+    # we don't need much out of setuptools, but the __init__.py stuff does
+    # need pkg_resources . We use >=11.3 here because that's what
+    # "cryptography" requires (which is a sub-dependency of TLS-using
+    # packages), so there's no point in requiring less.
+    "setuptools >= 28.8.0",
```

Comment:
I know this is just moving code around, but that comment looks .. outdated at best.